src/style.css

```
@@ -1 +1,56 @@
-/* Styles go here */
+.header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  position: absolute;
+  height: 60px;
+  left: 0;
+  top: 0;
+  width: 100%;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.nav {
+  padding: 0;
+  width: 100%;
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  list-style: none;
+  align-items: center;
+  justify-content: flex-end;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.nav_item {
+  padding: 0;
+  margin: 22px 10px;
+}
+
+.nav_link {
+  padding: 22px 0;
+  display: flex;
+  color: rgba(0, 0, 0, 1);
+  text-decoration: none;
+}
+
+.nav_link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active:active::after {
+  content: "";
+  position: absolute;
```

```suggestion
  position: absolute;
```
Відносно чого позиціонується цей after? Його батьківський компонент ніяк не спозиціонований